easybuild/framework/easyconfig/templates.py

```diff
@@ -203,8 +208,17 @@ def template_constant_dict(config, ignore=None, skip_lower=True):
     return template_values
 
 
-def template_documentation():
+def template_documentation(output_format):
```

Comment:
set a default to TXT, so you don't have to change it everywhere it's not passed on yet.